tests/test_experiment.py

```diff
@@ -77,11 +77,6 @@ def test_ensemble_no_name():
         )
 
 
-def test_bad_exp_path() -> None:
-    with pytest.raises(NotADirectoryError):
-        exp = Experiment("test", "not-a-directory")
```

Comment:
Why was this test removed?

Reply:
This is something we want to support, but was deemed out of scope for this PR. Reverting this change